src/Compilation.zig

```diff
@@ -800,7 +800,10 @@ pub fn create(gpa: Allocator, options: InitOptions) !*Compilation {
 
         const tsan = options.want_tsan orelse false;
         // TSAN is implemented in C++ so it requires linking libc++.
-        const link_libcpp = options.link_libcpp or tsan;
+        // libunwind is implemented in C++ so it requires linking libc++
+        const link_libcpp = options.link_libcpp or
+            (options.link_libunwind or target_util.osRequiresLibC(options.target) and target_util.libcNeedsLibUnwind(options.target)) or
+            tsan;
```

Comment:
```suggestion
        const link_libcpp = options.link_libcpp or tsan or link_libunwind;
```

Reply:
Leaving Compilation.zig untouched since changing src/libunwind.zig seems to fix the failing build of the zig port of serenityos.